src/subscription_manager/cache.py

```diff
@@ -96,6 +96,9 @@ def delete_cache(cls):
     def _cache_exists(self):
         return os.path.exists(self.CACHE_FILE)
 
+    def exists(self):
```

Comment:
this is a quick solution; a slightly bigger effort would be to simply rename `_cache_exists()` as `exists()`